src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -142,14 +183,15 @@ public EditPersonDescriptor(EditPersonDescriptor toCopy) {
             setPhone(toCopy.phone);
             setEmail(toCopy.email);
             setAddress(toCopy.address);
+            setEventTags(toCopy.eventIndexTags);
```

Comment:
Do you think the method name should be changed to this? I think using `setEventTags` can be misleading because there is a `setTags` method too.

```suggestion
            setEventIndexTags(toCopy.eventIndexTags);
```